.github/dependabot.yml

```diff
@@ -0,0 +1,39 @@
+version: 2
+updates:
+  - package-ecosystem: "docker"
+    directory: "/"
```

Comment:
What are we capturing in root that isn't captured by the other directories?

Reply:
Hah! I was just reading about it and confirming if this should be here or not. Docker compose is natively supported by Docker, so I thought that perhaps this dependabot could be used to update our docker compose files. I've confirmed that isn't the case yet. They have some open tickets about it, so maybe one day.

I'll remove this.